replication/main.go

```diff
@@ -0,0 +1,116 @@
+package main
```

Comment:
It's convention in Go packages to put these in a `cmd` folder. Think we should keep that convention